### ISSUE TITLE
Details on statistical model behind NER  

### ISSUE THREAD

**Issue by lukeskywalkre** [explosion/spaCy#2575]
The documentation on the actual models behind NER-tagging and training is rather sparse: 

> spaCy's models are statistical and every "decision" they make – for example, which part-of-speech tag to assign, or whether a word is a named entity – is a prediction. This prediction is based on the examples the model has seen during training. To train a model, you first need training data – examples of text, and the labels you want the model to predict. This could be a part-of-speech tag, a named entity or any other information.

Is there a paper or other type of documentation available that provides more detailed information on the models and, particularly, how they are trained? 

Many thanks!

**Comment by ines** [comment]
Yes, the paragraph you're quoting is really only the "spaCy models 101". 

You're probably looking for this section, which describes the model architecture in detail: https://spacy.io/api/#section-nn-model

There's also this video (~1h) of  @honnibal explaining the spaCy v2.x models in more detail:
https://www.youtube.com/watch?v=sqDHBH9IjRU

If you're interested in the hash embeddings, see this comment (which is also an excerpt from the upcoming book):
https://support.prodi.gy/t/can-you-explain-how-exactly-hashembed-works/564/2

For more details on the pre-trained models we distribute (including the data they were trained on and accuracy figures) see the models directory, e.g.:
https://spacy.io/models/en

**Comment by lukeskywalkre** [comment]
Thank you very much!
